### PR TITLE
Feature/intial tests

### PR DESCRIPTION
**Pull request by jonathanGR3450**
configuring docker to environment test

### CODE REVIEW

File: test_project/settings.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
+"""
+Django settings for test_project project.
```
Comment (javierriveros) [inline test_project/settings.py:2]: Why do you have test_app/ and test_project/? Shouldn't be just one?

File: requirements/requirements-testing.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#test dependecies
+coverage
+tox
+pytest>=4.6.1,<4.7
+pytest-django==3.9.0
+pytest-cov==2.9.0
+pytest-timeout==1.3.4
```
Comment (javierriveros) [inline requirements/requirements-testing.txt]: You probably don't need pytest because you're using django test module

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    volumes:
+      - ./pgdata:/var/lib/postgresql/data
+    ports:
+      - "5433:5432"
+
+  web:
+    build: .
+    container_name: djangoplicity-actions-web
+    command: python manage.py migrate
+    command: python manage.py runserver 0.0.0.0:8000
```
Comment (javierriveros) [inline docker-compose.yml]: You can use `command: ['./scripts/command-dev.sh']`, that's why that file is included

File: requirements/requirements-essential.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+django==1.11.29
+# dependeces setup
```
Comment (javierriveros) [inline requirements/requirements-essential.txt]: Small typo here.

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
+    "django.snippets.exclude": [
+    
+        "cms",
+        "wagtail"
+    ],
+    "python.venvFolders": [
+        "/home/jonathan/Escritorio/pasantia/repositorios/venv/django1/bin/python2.7"
+    ],
+}
```
Comment (EdisonArango) [inline .vscode/settings.json]: We don't need this.

File: docker-compose.yml (hunk 3 above)
Comment (EdisonArango) [reply docker-compose.yml]: Right, also because multiple commands here doesn't work.

**Comment by EdisonArango** [comment]
Please remove the htmlcov directory and ignore it in the gitignore, and you have received feedback from Javier since one week ago and you haven't addressed them.

**Pull request merged**